avalara_salestax/__openerp__.py

```diff
@@ -64,8 +64,5 @@
         "images/main_screenshot.png",   
         "images/kranbery_logo.png",     
     ],
-    'demo': [
```

Comment:
Removing the demo config as it messes up tests. Tests only run on init which means the demo config overwrites the custom authentication data in my test database.